### PR TITLE
Wishlist 

### PR DESCRIPTION
**Pull request by dikshadhote**
Following functionalities have been added:
1. Add to wishlist
2. Remove from wishlist
3. Add only unique cards in wishlist and cart
4. if same card added in cart increase quantity
5. Empty wishlist cart page 
image

### CODE REVIEW

File: src/components/navbar/Navbar.js [hunk 1]
```diff
@@ -3,9 +3,12 @@ import "./../../App.css";
...
 import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
 export default function Navbar() {
   const { stateCart } = useCart();
   const { cart } = stateCart;
+  const { stateWishlist } = useWishlist();
+  const { wishlist } = stateWishlist;
   return (
     <div>
       {" "}
```
Comment (JantuDeb) [inline src/components/navbar/Navbar.js]: You can remove empty jsx expression

File: src/components/navbar/Navbar.js [hunk 2]
```diff
@@ -28,9 +31,16 @@ export default function Navbar() {
...
-              favorite
-            </span>
+            <div className="icon-container">
+              <span className="material-icons mr-1 ml-1" title="wishlist">
+                favorite
+              </span>
+              {wishlist.length > 0 ? (
+                <span className="badge orange-bg position-mail-badge small-badge">
+                  <small className="number">{wishlist.length}</small>
+                </span>
```
Comment (JantuDeb) [inline src/components/navbar/Navbar.js]:
```
wishlist.length > 0 &&  (
                <span className="badge orange-bg position-mail-badge small-badge">
                  <small className="number">{wishlist.length}</small>
                </span>)


```

**Review by JantuDeb — APPROVED** [review]
Overall it is well structured code, left few comments do check it out.

**Review by tusharkandpal — APPROVED** [review]
code is clean & organized.

Suggestion: can use barrel export.

**Pull request merged**